sensor/common/sensor/central_receiver_impl.go

```diff
@@ -39,19 +39,23 @@ func (s *centralReceiverImpl) receive(stream central.SensorService_CommunicateCl
 	for {
 		select {
 		case <-s.stopper.Flow().StopRequested():
+			log.Infof("Central receiver stop flow requested")
```

Comment:
I thought this is a typo and wanted to correct `receiver` -> `received`.
Maybe:

```suggestion
			log.Infof("Central receiver got stop flow requested")
```